apps/qa/src/shared/components/sidebar.py

```diff
@@ -43,7 +43,9 @@ def data_selection(
                     )
         case "Published":
             label = "Select a version"
-            options = publishing.get_published_versions(product)
+            options = publishing.get_published_versions(
```

Comment:
Ahh - makes sense that the QA app would have some cases where we'd want latest. Thank you for catching!